### PR TITLE
Update the eval tutorial

### PR DESCRIPTION
**Pull request by TuanaCelik**

### CODE REVIEW

File: index.toml [hunk 1]
```diff
@@ -377,12 +377,12 @@ haystack_2 = true
 dependencies = ["accelerate", "sentence-transformers", "datasets"]
 
 [[tutorial]]
-title = "Model-Based Evaluation of RAG Pipelines"
-description = "Learn how to evaluate your RAG pipelines using some of the model-based evaluation frameworkes integerated into Haystack"
+title = "Evaluating RAG Pipelines"
+description = "Learn how to evaluate your RAG pipelines using statistical and model-based evaluation  metrics"
```
Comment (bilgeyucel) [inline index.toml:381]:
```suggestion
description = "Learn how to evaluate your RAG pipelines using statistical and model-based evaluation metrics"
```

**Comment by julian-risch** [comment]
<p>AnswerFaithfulnessEvaluator or AnswerHallucinationEvaluator are alternatives.</p>

---

 <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/318/discussion/#comment-2084794030'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by TuanaCelik** [comment]
<p>Because that's the only live one right now. Will change when released.</p>

---

 <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/318/discussion/#comment-2084793920'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by TuanaCelik** [comment]
<p>right now yes, will change when released</p>

---

 <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/318/discussion/#comment-2084793980'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by TuanaCelik** [comment]
<p>thanks 🙏</p>

---

 <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/318/discussion/#comment-2084794001'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by julian-risch** [comment]
@TuanaCelik The only change needed in this tutorial that is caused by the [PR](https://github.com/deepset-ai/haystack/pull/7621) I just merged is that FaithfulnessEvaluator's input parameter `responses` was renamed to `predicted_answers`. And if you think we should rename the component we can still do it, just let me know.

**Comment by TuanaCelik** [comment]
> @TuanaCelik The only change needed in this tutorial that is caused by the [PR](https://github.com/deepset-ai/haystack/pull/7621) I just merged is that FaithfulnessEvaluator's input parameter `responses` was renamed to `predicted_answers`. And if you think we should rename the component we can still do it, just let me know.

Hey @julian-risch - thanks for the info. I don't have strong opinions on the component naming. Imo, 'faithfulness' is widely used at this point. I'll defer to you guys to make the final call here. I'm ok with either

**Review by bilgeyucel — APPROVED** [review]

**Pull request merged**